packages/helium-vote-service/src/index.ts

```diff
@@ -2,6 +2,7 @@ import cors from "@fastify/cors";
 import fastifyStatic from "@fastify/static";
 import { organizationKey } from "@helium/organization-sdk";
 import { createMemoInstruction } from "@solana/spl-memo";
+import { DuneClient } from "@duneanalytics/client-sdk";
```

Comment:
I wonder if it makes more sense to sync this into pg sink as we do other derived data.

Reply:
We'd have to do a transaction-based sink. So the existing account sink code wouldn't work. Not worth the effort imo unless we start doing more analytics

Reply:
yeah that's fair. this is a good start though.